utils/search_tools.py

```
@@ -51,54 +60,60 @@ def build_search_args(self) -> str:
+            "",
+            name,
+            flags=re.IGNORECASE,
+        )
         # Remove unwanted whitespaces
-        name = re.sub(r'\s+', ' ', name)
+        name = re.sub(r"\s+", " ", name)
         # Remove leading and trailing whitespaces
         name = name.strip()
 
-        logger.debug(f'Normalized Name: {name}')
+        logger.debug(f"Normalized Name: {name}")
```

Ruff (0.15.0)

[warning] Logging statement uses f-string

(G004)